src/godot-tools.ts

```diff
@@ -11,7 +11,6 @@ export class GodotTools {
 
 	private context: vscode.ExtensionContext;
 	private client: GDScriptLanguageClient = null;
-	private workspace_dir = vscode.workspace.rootPath;
```

Comment:
I know nothing about vscode extensions. I originally wanted to call `get_configuration` here, but I decoded to move the `get_configuration` into each method call below because I wanted to make sure we had the latest value in `settings.json`.

Reply:
Ah, i found it here: https://code.visualstudio.com/api/references/vscode-api#workspace
```
onDidChangeConfiguration: Event<ConfigurationChangeEvent>
```